Desktop/Main.cs

```diff
@@ -22,6 +22,7 @@ private void Main_Load(object sender, EventArgs e)
         {
             PrepareTransactionListView();
             UpdateTransactionList();
+            IncomeAndOutcomeCalculation(_transactionList);
```

Comment:
It's not necessary to pass a transaction list because it is a property of a class.